### PR TITLE
Adding USA2021's coordinates

### PR DESCRIPTION
**Pull request by USA2021**
@githubteacher what do you think?
this pull request closes#940.

### CODE REVIEW

**Comment by nhutly** [comment]
This's great

File: _pins/USA2021.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+githubHandle: USA2021
```
Comment (crichID) [inline _pins/USA2021.json]: @USA2021 you  just need to  add three dashes `---` on the first line of this file and your checks should pass!  If you need  help making the change, please check out this video: https://vimeo.com/201011288

**Comment by crichID** [comment]
@USA2021 you had a trailing  space hiding at the end  of one of your lines. I removed it in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/12224/commits/382e13ffb73de4ac6fd9de64f05632e13b9c015e.  Once the tests are  completed,  it should be ready for you to Merge.

**Comment by shaylaprice** [comment]
@USA2021 Great job!

**Comment by Justme764** [comment]
@USA2021  you got the green check, I'm still trying to figure that out but yours looks great.

**Comment by Justme764** [comment]
@USA2021  you got the green check, I'm still trying to figure that out but yours looks great.

**Pull request merged**